deps/openssl/config/archs/aix-gcc/opensslconf.h

```diff
@@ -235,7 +235,7 @@ extern "C" {
    even newer MIPS CPU's, but at the moment one size fits all for
    optimization options.  Older Sparc's work better with only UNROLL, but
    there's no way to tell at compile time what it is you're running on */
-
+ 
```

Comment:
I was not sure and thought it would be better to be safe than sorry and get feedback on the PR first. Let me know and I'll remove that commit. Thanks